brambling/templates/brambling/event/order/_use_discount_js.html

```diff
@@ -1,6 +1,6 @@
 <script>
     $(function(){
-        $('#discountCodeButton').on('click', function(e){
+        $('#discountForm').on('submit', function(e){
             e.preventDefault();
             var discountCodeButton = $(this),
```

Comment:
This will need to be corrected. In the old version of the code, `this` referred to the `<button>`, but in the new version `this` refers to the `<form>`. The button will have to be identified some other way. (`$(this).find('button')` might work.)